database/dbSeed.js

```diff
@@ -1,6 +1,52 @@
-const db = require('../index.js');
-let property_count = 100;
+const db = require('./index.js');
 
-const seedPropertyDetails = funciton(){
-  
-}
\ No newline at end of file
+const booleanFillers = () => {
+  return Math.random() < 0.5;
+};
+
+const maxStay = (min) => {
+  return Math.floor(Math.random() * (15 - min) + min);
+};
+
+const seedPropertyDetails = function() {
+  let propertyCount = 100;
+  let propType = {
+    'Apartment': ['Condominium', 'Serviced apartment', 'Loft'],
+    'House': ['Villa', 'Townhouse', 'Cottage', 'Bungalow', 'Cabin'],
+    'Secondary Unit': ['Guest Suite', 'Guesthouse', 'Farm stay'],
+    'Unique Space': ['Barn', 'Tiny house', 'Farm stay', 'Camper/RV', 'Boat', 'Campsite', 'Earth house'],
+    'Bed and breakfast': ['Farm stay', 'Nature lodge'],
+    'Boutique hotel': ['Boutique hotel', 'Hostel', 'Resort', 'Serviced apartment']
+  };
+  let spaceAvail = ['Private room', 'Entire place', 'Shared room'];
+  let adjectives = ['huge', 'extraordinary', 'amazing', 'pleasant', 'cozy', 'enjoyable', 'delightful', 'charming', 'beautiful', 'fine', 'pleasing', 'elegant', 'alluring', 'lovely', 'attractive', 'chic', 'enchanting', 'irresistible', 'stimulating' ];
+  let attraction = ['park', 'amusement park', 'beach', 'campsite', 'shopping outlet', 'farmers market', 'resort', 'famous trail', 'night life plaza', 'winery', 'golf course', 'dog park', 'rose garden', 'museum', 'zoo', 'recreation area'];
+
+  while (propertyCount > 0) {
+    let typeArr = Object.keys(propType);
+    let type = typeArr[Math.floor(Math.random() * 6)];
```

Comment:
@pauly-van A little unclear what the 6 value is for, is this because of the length of propType? If so maybe it'd be more clear to use propType.length.

Reply:
it looks like you're trying to select a random index in typeArr which has a length of 4, while Math.random()*6 is returning a number between 0 and 5. Seems like it may be good to go off of typeArr.length instead of 6.